### PR TITLE
DIV-6209-call-pba-creditaccountapi-to-initiate-payment-revert

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DIV-6209

### Change description ###

Updating config to point to Demo to COS PR-1054

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/demo/common/divorce/div-cos.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: divorce
   annotations:
     flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
+    flux.weave.works/tag.java: glob:pr-1054-*
```
Comment (yarpo) [inline k8s/demo/common/divorce/div-cos.yaml:9]: why?

**Review by yarpo — CHANGES_REQUESTED** [review]
I thought PR-1054 is merged, tested on aat.

Deployed on aat, demo and prod. Works for config 1 and config 2.

If so - we don't need this update anymore.

**Review by yarpo — APPROVED** [review]
OK, I'm it's not breaking anything and apparently this can help testing.

File: k8s/demo/common/divorce/div-cos.yaml (hunk 1 above)
Comment (madjava) [reply k8s/demo/common/divorce/div-cos.yaml:9]: To get PR build to demo. Not deployed on AAT yet

**Pull request merged**